framework/include/utils/RankTwoTensor.h

```diff
@@ -405,6 +405,14 @@ class RankTwoTensor
   /// returns this_ij * b_ijkl
   RankTwoTensor initialContraction(const RankFourTensor & b) const;
 
+
+  /// rank two permutation tensor (also known as Levi-Civita symbol):
+  /* eps(i1, i2, ..., iD) =
+  *   +1 if (i1, i2, ..., iD) is an even permutation of (0, 1, ..., D)
+  *   -1 if (i1, i2, ..., iD) is an odd permutation of (0, 1, ..., D)
+  /*    0 otherwise */
+  static int eps(unsigned int i, unsigned int j);
```

Comment:
Are you asking yourself a question? 😉 

The reason anything might need to be static is because you don't have an object you are calling this method on. You are just using it as a free function. As long as the function in question doesn't touch any member data on the class that it's defined in, it can be made static. This is actually desired and nice for many types of utility functions. I don't know if my answer is what you are looking for though.